.travis.yml

```diff
@@ -21,66 +21,67 @@ env:
     # Set default python version to avoid repetition later
     - BUILD_DOCS=false
     - PYTHON_VERSION=2.7
-    - MAIN_CMD="./testsuite/MDAnalysisTests/mda_nosetests --with-coverage --cover-package MDAnalysis --processes=2 --process-timeout=400 --with-memleak --with-timer --timer-top-n 50 --no-open-files"
-    - SETUP=minimal
+    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
```

Comment:
Out of curiosity, why did it become necessary to specifically call python rather than just calling the script?